cmd/intervalContacts/intervalContacts.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/bed/bedpe"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/interval"
+	"log"
+)
+
+func intervalContacts(bedpeFile string, inFile string, contactOutFile string) {
+	var inIntervals = make([]interval.Interval, 0)
+	var currOverlaps []interval.Interval
+	var err error
+
+	out := fileio.EasyCreate(contactOutFile)
+
+	contactRecords := bedpe.Read(bedpeFile)
+	inChan := interval.GoReadToChan(inFile)
+
+	for i := range inChan {
+		inIntervals = append(inIntervals, i)
+	}
+	inTree := interval.BuildTree(inIntervals)
+
+	for _, i := range contactRecords {
+		currOverlaps = interval.Query(inTree, i.A, "any")
+		if len(currOverlaps) > 0 {
+			bed.WriteBed(out, i.B) // if anything in the input set overlaps i.A, write i.B to the output.
+		}
+		currOverlaps = interval.Query(inTree, i.B, "any")
+		if len(currOverlaps) > 0 {
+			bed.WriteBed(out, i.A) // and vice versa
+		}
+	}
+
+	err = out.Close()
+	exception.PanicOnErr(err)
+}
+
+func usage() {
+	fmt.Print("intervalContacts - Returns all regions that contact input genomic regions. " +
```

Comment:
I think it would be useful to list the types of files gonomics can support with our interval package. I see based on test files we can handle .bed and .vcf. Are there other interval file types we can handle too?